src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null
```

Comment:
```suggestion
  pressedKey: string | null
```
According to the task description, it should be called like this
![image](https://user-images.githubusercontent.com/96409563/222362183-4ae1380f-8195-4097-aa45-9ff516878c43.png)